fansi/shared/src/main/scala/fansi/Fansi.scala

```diff
@@ -742,3 +761,35 @@ private[this] final class Trie[T](strings: Seq[(String, T)]){
   }
 }
 
+
+
+trait TrueColor extends Category {
+  /**
+    * 38 : front color
+    * 48 : background color
+    */
+  val zindex : Int
```

Comment:
Honestly I prefer `trueColorCode`. The spec may call it `selectedGraphicRendition` but I doubt anyone will know what in the world that means.